test/tests.zig

```diff
@@ -579,6 +579,20 @@ const test_targets = blk: {
             .link_libc = true,
         },
 
+        .{
+            .target = std.Target.Query.parse(.{
+                .arch_os_abi = "riscv32-linux-none",
+                .cpu_features = "baseline-d-f",
+            }) catch @panic("OOM"),
```

Comment:
I know this was copied from the existing entries but the panic message is nonsense, that would be a target string parse failure rather than OOM. `catch unreachable` is fine probably.

Reply:
I'll replace those with `catch unreachable` next time I touch the file.